Lesson_17/home_work/script.js

```diff
@@ -0,0 +1,680 @@
+				break;
+			case 'г.': return 'грам';
+				break;
+			case 'кг.': return 'кілограм';
+				break;
+			case 'мм.': return 'міліметр';
+				break;
+		}
+	}
+	
+	for (var i = 0; i < values.length; i++){
+		
+		all += names[i].toUpperCase() + ' до ' + getFullName(ourNames[i]).toUpperCase() + ': \n';
+		
+		for (var j = 1; j < 11; j++){
+			console.log( j + ' ' + names[i] + ' = ' + j * values[i] + ' ' + ourNames[i] );
+			all +=  j + ' ' + names[i] + ' = ' + j * values[i] + ' ' + ourNames[i] + ' \n ';
+		}
+		
+		all += '\n';
+	}
+	alert(all);
+}
+// Task 6
+function getAcountValue(){
+	
+	var sum = parseInt(prompt('', 'Enter the sum of your money')),
+		years = parseInt(prompt('', 'Enter how many years you`d like to keep money on account'));
+	
+	for (var i = 0; i < years; i++){
+		sum *= 1.03;
+	}
+	sum = Math.round(sum);
+	alert(sum);
+	
+}
+// Task 7
+function getTheBiggest(){
+	
+	var numbers = prompt('', 'Enter ten pairs of numbres like in example: X, X; and then press "OK"'),
+		numberArr = numbers.split('; ');
+	
+	for (var i = 0; i < numberArr.length; i++){
+		var result = numberArr[i].split(', ');
+		
+		if (parseInt(result[0]) > parseInt(result[1])){
+			alert('The biggest number is ' + result[0]);
+		} else {
+			alert('The biggest number is ' + result[1]);
+		}
+		
+	}
+	
+}
+// Task 8
+function showDividedOn(){
+	var number = new Array;
+	
+	for (var i = 20; i < 51; i++){
+		if ((i % 3 === 0) && (i % 5 !== 0)){
+			number.push(' ' + i);
+		}
+		
+	}
+	
+	alert(number);
+}
+// Task 9
+function showDividedOnSeven(){
+	var number = new Array;
+	
+	for (var i = 35; i < 88; i++){
+		if ((i % 7 === 1) || (i % 7 === 2) || (i % 7 === 5)){
+			number.push(' ' + i);
+		}
+		
+	}
+	
+	alert(number);
+}
+// Task 10
+function showSumDividedOn(){
+	var result = 0;
+	
+	for (var i = 1; i < 51; i++){
+		if ((i % 5 === 0) || (i % 7 === 0)){
+			result += i;
+		}
+		
+	}
+	
+	alert(result);
+}
+// Task 11
+function printNumbers(){
+	var number = new Array;
+	
+	for (var i = 10; i < 100; i++){
+		if ((i % 4 === 0) && (i % 6 !== 0)){
+			number.push(' ' + i);
+		}
+		
+	}
+	
+	alert(number);
+}
+// Task 12
+function getMultipleOf(){
+	var number = new Array,
+		result = 0;
+	
+	for (var i = 10; i < 100; i++){
+		if ((i % 13 === 0)){
+			result = i * i;
+			number.push(' ' + result);
+		}
+	}
+	alert(number);
+}
+// Task 13
+function getSumOf(){
+	var result = 0;
+	
+	for (var i = 100; i < 201; i++){
+		if (i % 17 === 0){
+			result += i;
+		}
+		
+	}
+	
+	alert(result);
+}
+// Task 14
+function getSumOfPow(){
+	var lastNumber = parseInt(prompt('', 'Enter the last number of set')),
+		result = 0;
+	
+	for (var i = 1; i < lastNumber; i++){
+		result += i * i;
+	}
+	
+	alert(result);
+}
+// Task 15
+function getPowOfValues(){
+	var a = parseFloat(prompt('', "Enter value for A")),
+		b = parseFloat(prompt('', "Enter value for B"));
+	
+	if ((a % parseInt(a) > 0) && (b < 0)){
+		alert('b не может быть отрицательным');
+		return getPowOfValues();
```

Comment:
what does this code do, if we have entered bad values, it calls itself, bad idea